Source/Core/Core/HW/DSPHLE/UCodes/AXVoice.h

```diff
@@ -81,17 +81,50 @@ union AXBuffers
 };
 
 // Read a PB from MRAM/ARAM
-void ReadPB(u32 addr, PB_TYPE& pb)
+void ReadPB(u32 addr, PB_TYPE& pb, bool has_lpf = true)
```

Comment:
Slight cosmetic preference: I would prefer if ReadPB/WritePB took the CRC as input and then computed HasLpf internally.

It also makes little sense to me to have a default value here -- that's just going to hide bugs.